velox/connectors/hive/HiveConfig.cpp

```diff
@@ -149,7 +149,10 @@ bool HiveConfig::isFileColumnNamesReadAsLowerCase(const Config* session) const {
 }
 
 bool HiveConfig::isPartitionPathAsLowerCase(const Config* session) const {
-  return config_->get<bool>(kPartitionPathAsLowerCaseSession, true);
+  if (session->isValueExists(kPartitionPathAsLowerCaseSession)) {
```

Comment:
It is confusing to me that Config* variable is called session.

@kewang1024 Ke, are you familiar with this code? Would you help review?